soc/arm/st_stm32/stm32h7/sections.ld

```diff
@@ -0,0 +1,17 @@
+/*
+ * Copyright (c) 2020 Mario Jaun
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#if DT_NODE_HAS_STATUS(DT_NODELABEL(sram3), okay) && DT_NODE_HAS_STATUS(DT_NODELABEL(mac), okay)
```

Comment:
I would prefer to name this region `no_cache_no_mpu_sram` and add it accordingly to KConfig of stm32